apis/fluentbit/v1alpha2/fluentbit_types.go

```diff
@@ -107,6 +107,8 @@ type FluentBitSpec struct {
 	SchedulerName string `json:"schedulerName,omitempty"`
 	// Optional duration in seconds the pod needs to terminate gracefully. Value must be non-negative integer.
 	TerminationGracePeriodSeconds *int64 `json:"terminationGracePeriodSeconds,omitempty"`
+	// Storage for wasm file. You will use it if wasm filter is enabled.
+	WasmFile corev1.VolumeSource `json:"wasmFile,omitempty"`
```

Comment:
[Volumes  and VolumesMounts](https://github.com/fluent/fluent-operator/blob/master/apis/fluentbit/v1alpha2/fluentbit_types.go#L70-L72) can be used to mount wasm files. Does it need to add additional fields?

Reply:
I think it doesn't need additional fields, just specify the hostPath of the wasm file. 
Like this:
```
wasmFile:
    hostPath:
      path: /etc/wasm/  
```